### ISSUE TITLE
I can't get a working reactUI.example project anymore

### ISSUE THREAD

**Issue by adriahuertas** [sibvisions/reactUI.example#5]
I'm not able to get a working reactUI.example with the latests repositories by following the dev setup instructions anymore. What I do:

- Git clone both the reactUI and reactUI.example repositories and npm install in both directories.
- In reactUI directory: npm link
- In reactUI.example directory: npm link @sibvisions/reactui
- In reactUI directory: npm link ../reactUIExample/node_modules/react  (By the way, there's a typo in the README, the name of the directory is not reactUIExample but reactUI.example)
- In reactUI.example: npm start

At this point I got a working project before, but now I'm getting the "Can't resolve '@sibvisions/reactui' [...]" in the files of reactUI.example. I've tried to run both npm run bundle and devbundle in the reactUI directory as suggested in the README, but that didn't fix the error.

I'm quite new to npm, so I might be doing something wrong. Any hint?

**Comment by maximilianweidenauer** [comment]
Ok, so the thing is you **only** need to clone the reactUI project if you want to make changes to it. 

I think in your case it is best if you don't use the linking and bundling steps. I think for you it is sufficient if you just use the reactUI.example with the reactUI version 1.0.43 (npm install @sibvisions/reactui) which can be downloaded from the npm registry, as I remember from a previous ticket you have already done that.
It's either that **OR** bundling not both! I would recommend for you to unlink the reactui and react (`npm unlink @sibvisions/reactui`, `npm unlink ../reactUIExample/node_modules/react`).

What your problem probably is, is that when you are using the linking/bundling method, the project structure is a bit different than when you are using the npm package version.
Linking/Bundling imports have an extra `dist` folder which has to be accessed for the imports to work.
E.g:
This is an import when you are using the npm package version (npm install @sibvisions/reactui): `import { ScreenWrapper, useAPI } from "@sibvisions/reactui";`

This is an import when you are using linking/bundling: `import { ScreenWrapper, useAPI } from "@sibvisions/reactui/dist";`

**Comment by maximilianweidenauer** [comment]
I should update the README regarding that matter but I haven't had the time for that yet

**Comment by adriahuertas** [comment]
Ok, I tried before to use only the reactUI.example repository and I've got an error so I've tried the other approach without paying attention to the error. I've tried again now and here is the error: 
![imagen](https://user-images.githubusercontent.com/91001941/202447219-91f50bb7-15c4-4ea0-99c9-a1acea144f4d.png)
It seems that there is unupdated import. So I've replaced "@sibvisions/reactui/dist" with "@sibvisions/reactui" in the CustomAppWrapper.tsx file as you've just mentioned and it's working now. Thanks,
